px-data-grid-cell-content-wrapper.html

```diff
@@ -53,6 +53,17 @@
 
           this.addEventListener('mouseover', this._fireCellHoverEvent);
           this.addEventListener('mouseout', this._fireCellUnhoverEvent);
+          document.addEventListener('editing-item-changed', (event) => {
```

Comment:
Should this be called editted-item-changed (also maybe property should be called editted-item)